automation/vars/default.yaml

```diff
@@ -0,0 +1,46 @@
+---
+vas:
+  hci:
+    repo: https://github.com/openstack-k8s-operators/architecture.git
+    branch: main
+    stages:
+    - path: examples/va/hci/control-plane/nncp
+      validation: "oc wait nncp --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=60" 
+      values:
+      - name: network-values
+        src_file: network_values.yaml
+        dest_file: values.yaml
+
+    - path: examples/va/hci/control-plane
+      validation: "oc wait osctlplane controlplane --for condition=Ready --timeout=600s"
+      values:
+      - name: network-values
+        src_file: network_values.yaml
+        dest_file: nncp/values.yaml
```

Comment:
IIRC, `dest_file` is to override the values.yaml in Andrew's PoC. Would it be possible to point the file that should be created by `kustomize build` instead? Doing so, we'd be able to get an easy path to automation, with the `src_file` pointing to the existing values.yaml, and `dest_file` being the output needed by the `oc apply`.